src/app/Navbar/Navbar.module.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please remove css file as this belongs to a another ticket.

Reply:
sorry im going to that :(